lib/adal-angular.js

```diff
@@ -135,7 +132,8 @@
                                     if (!$location.$$html5 && loginStartPage.indexOf('#') > -1) {
```

Comment:
Please move the log statement and this if statement inside the if block for adal.popUp